### DISCUSSION TITLE
502 when attempting to delete an image from ghcr

### DISCUSSION THREAD

**Discussion by TeddiO** [community/community#24639] [Apps, API and Webhooks]
<p>Hi there, over the last few days when attempting to delete an image from the GHCR 99% of the time I’ll get a HTTP response of 502. Even worse it seems to be a coin toss as to whether or not the image actually gets deleted (sometimes it will, sometimes it won’t).</p>
<p>I can’t find anything in the docs that suggests there have been any updates to the requirements and other similar API calls I perform to fetch the image version work without issue.</p>
<p>At bare minimum, this is what I’m trying to attempt:</p>
<pre><code class="lang-bash">curl \
-X DELETE \
-H "Accept: application/vnd.github.v3+json" \
-H "authorization: Bearer &lt;token&gt;" \
https://api.github.com/orgs/&lt;org&gt;/packages/container/&lt;container&gt;/versions/&lt;version&gt;
</code></pre>
<p>Any thoughts as to what I may be missing or doing wrong? Thanks!</p>

**Comment by TeddiO** [comment, marked as answer]
<p>And it appears the endpoint is working correctly once again! Thanks to the Engineer who has resolved this! :grin:</p>

**Comment by TeddiO** [comment]
<p>Seems I was a bit too enthusiastic, the issue is back :frowning_face:</p>

**Comment by mmenozzi** [comment]
<p>I have this issue too…</p>
